### ISSUE TITLE
Datatable: cannot change column sort direction using keyboad

### ISSUE THREAD

**Issue by arnodemer** [primefaces/primereact#5594]
### Describe the bug

Unable to change datatable column's sort direction using keyboard (but click on header works).

This is a regression since v9.6.5-snapshot, and an accessibility issue.

### Reproducer

https://primereact.org/datatable/

### PrimeReact version

10.2.1

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Browser(s)

Opera One(version: 105.0.4970.34),  Chroem Version 120.0.6099.71, ...

### Steps to reproduce the behavior

1. open: https://primereact.org/datatable/
2. select: primreact 10.2.1 on the right
3. open features of Datatable component, go to sort feature.
4. set focus on column header, hit Enter key
=> the sort direction of column doesn't changed

Note: select v9.6.5-snapshot:  Enter key will works to change column sort order.

### Expected behavior

Datatable column sort direction must be changed hen user hit enter key
(Treetable component works)

**Comment by melloware** [comment]
All keyboard and Accessibility issues are being addressed by PrimeTek in an upcoming release.

**Comment by melloware** [comment]
You can see on PrimeVue what PrimeReact will be getting: https://primevue.org/datatable/#accessibility

**Comment by arnodemer** [comment]
@melloware hope this will be fixed in primereact v11, thanks.

**Comment by melloware** [comment]
Please try 10.9.1 and I will reopen if this is not fixed.  But ENTER or SPACE should trigger the sorting on the focused header
